frontend/app/enrollment/components/enrollment-details-view.tsx

```diff
@@ -37,6 +37,8 @@ export default function EnrollmentDetailsView() {
                     <div className='col-span-3'>{patient?.name?.[0].text || "Unknown"}</div>
                     <div className='font-bold'>BSN:</div>
                     <div className='col-span-3'>{getBsn(patient) || "Unknown"}</div>
+                    <div className='font-bold'>Adres:</div>
```

Comment:
```suggestion
                    <div className='font-bold'>Adress:</div>
```

Reply:
Ah cheers, so very Dutch 😂 Will change it to Address

Reply:
I occasionally need corrections on my British-English spelling vs American spelling, so all good :P